README.md

```diff
@@ -49,6 +49,22 @@ $ python manage.py makemigrations # Creates one or more new files that must be c
 # Remember to format generated migrations! (using e.g. `make fixme`)
 ```
 
+**`poetry.lock` conflicts**
+
+If you have updated dependencies it's likely you might get conflicts in the Poetry lock file.
+This solution should resolve most conflicts quite well:
+
+```bash
+$ git rebase origin/master
+
+# If conflicts
+$ git checkout --thers poetry.lock
+
+$ poetry lock --no-update
+
+# The conflicts should be good resolved
```

Comment:
Could you change the wording to either "should be good" or "should be resolved", or something else?